pkg/openscap/openscap.go

```diff
@@ -158,6 +161,12 @@ func (s *defaultOSCAPScanner) Scan(mountPath string, image *docker.Image) error
 	s.image = image
 	s.imageMountPath = mountPath
 
+	s.imageArch = image.Architecture
+	if len(s.imageArch) == 0 {
+		log.Printf("Unable to identify image's architecture")
```

Comment:
@enoodle I apologize, I forgot this is in the pkg library. Please remove this line and the `"log"` import above. Other than that :+1: